src/views/Correction/BenCorrection.vue

```diff
@@ -100,19 +99,9 @@ export default class BenCorrection extends Mixins(CommonMixin, DateMixin, Filing
       // fetch business snapshot
       const businessSnapshot = await this.fetchBusinessSnapshot()
 
-      // *** FUTURE: remove this workaround
-      // set NR Number in snapshot since API doesn't return it yet and we need to
-      // know if this is a named company -- see ticket #13022
-      businessSnapshot.businessInfo.nrNumber =
-        this.correctionFiling.incorporationApplication.nameRequest.nrNumber
-
       // parse draft correction filing and business snapshot into store
       this.parseCorrectionFiling(this.correctionFiling, businessSnapshot)
 
-      // fetch and store corrected filing
-      const correctedFiling = await LegalServices.fetchFilingById(this.getBusinessId, this.getCorrectedFilingId)
-      this.setCorrectedFiling(correctedFiling)
```

Comment:
We no longer need to fetch the original filing! This is because we now compare with the current business state (snapshot), and we get the other metadata directly from the correction object.

Ditto in FmCorrection.vue.